src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IDelegateCreationExpression.vb

```diff
@@ -3502,6 +3686,176 @@ BC30469: Reference to a non-shared member requires an object reference.
             VerifyOperationTreeAndDiagnosticsForTest(Of UnaryExpressionSyntax)(source, expectedOperationTree, expectedDiagnostics)
         End Sub
 
+        <CompilerTrait(CompilerFeature.IOperation)>
+        <Fact()>
+        Public Sub DelegateCreation_ParenthesizedAddressOf_CType()
+            Dim source = <![CDATA[
+Imports System
+
+Public Class C
+    Public Sub M1()
+        CType(((AddressOf M2)), Action)'BIND:"CType(((AddressOf M2)), Action)"
```

Comment:
>((AddressOf M2)) [](start = 14, length = 16)

Do we have a test for a scenario when this parenthesized expression is implicitly converted?

Reply:
Added.

---
In reply to: [152436287](https://github.com/dotnet/roslyn/pull/23315#discussion_r152436287) [](ancestors = 152436287)